services/QuillLMS/app/controllers/teachers/classrooms_controller.rb

```diff
@@ -158,6 +157,13 @@ def transfer_ownership
     render json: {}
   end
 
+  private def fetch_classrooms_i_teach_cache
+    current_user.all_classrooms_cache(key: 'teachers.classrooms.classrooms_i_teach') do
+      @classrooms = current_user.classrooms_i_teach
+      @classrooms.sort_by { |c| c[:update_at] }
```

Comment:
I'm guessing this is an older bug, but should this be `:updated_at`?

Reply:
That's a good point. What about removing the sort_by altogether.

Reply:
The [docs]( https://ruby-doc.org/core-2.7.0/Enumerable.html#method-i-sort_by) mention that `sort_by` maps each value in the array to the block, but if the mappings are equal, the result is unpredictable.  Trying it out in the console a few times seems to return the original array.

Reply:
Seems reasonable.  I'll pull the sort and make sure tests pass, and we can cross our fingers that there's no weird gotchas in there.